sdk/servicebus/service-bus/package.json

```diff
@@ -60,7 +60,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"samples/**/*.{ts,js}\" \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"dist-esm/test/internal/**/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\"",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 600000 --full-trace \"dist-esm/test/internal/**/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\"",
```

Comment:
isn't this the timeout for a single test?

Reply:
Why do we need that long?

Reply:
yes, it is.  I want to reduce this as well.

Reply:
I could do 5 minutes. but let's use half for now and see if we need to further reduce in the future. I also noticed most of packages use 20 minutes.  This is only an issue when tests time out.  After most packages move to dev-tool it may make it easier to reduce the limite for them.